.flake8

```diff
@@ -0,0 +1,17 @@
+[flake8]
+ignore=E501
+max-complexity = 100
+exclude =
```

Comment:
I suffered some disappointments recently when he wanted to check *.js . It's why i have add *.js

For .pyc, i think i can delete it yes :)